src/index.tsx

```diff
@@ -679,25 +660,43 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
   };
 
   private renderModalize = (): React.ReactNode => {
-    const { modalStyle, adjustToContentHeight, keyboardAvoidingBehavior, alwaysOpen } = this.props;
+    const {
+      keyboardAvoidingOffset,
+      modalStyle,
+      keyboardAvoidingBehavior,
+      alwaysOpen,
+      panGestureEnabled,
+    } = this.props;
     const { isVisible, lastSnap, showContent } = this.state;
-    const enabled = isIos && adjustToContentHeight;
+    const enabled = isIos;
     const pointerEvents = alwaysOpen ? 'box-none' : 'auto';
 
+    const keyboardAvoidingViewProps: KeyboardAvoidingViewProps = {
+      keyboardVerticalOffset: keyboardAvoidingOffset,
+      behavior: keyboardAvoidingBehavior || 'padding',
+      enabled,
```

Comment:
```suggestion
      isIos,
```

Can you straight `isIos` and remove enabled variable